BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingViewController.swift

```diff
@@ -92,7 +92,6 @@ final class DigitalInvoiceOnboardingViewController: UIViewController {
         firstLabel.font = configuration.textStyleFonts[.title2Bold]
         firstLabel.textColor = GiniColor(light: .GiniBank.dark1, dark: .GiniBank.light1).uiColor()
         firstLabel.adjustsFontForContentSizeCategory = true
-        firstLabel.accessibilityValue = firstLabelText
```

Comment:
[Clarification] Was it removed by accident here?

Reply:
It was on purpose, I just found that accesibilityValue there and it will result in double reading of the text value on voice over. So I decided to remove it. :D